1001-2000/LIT1171Awdana.xml

```diff
@@ -53,15 +64,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <ptr target="bm:ContiRossini1941AwdaNagast"/>
                </bibl>
             </listBibl>
+            <listBibl type="secondary">
+               <bibl> 
+                  <ptr target="bm:Pietruschka2003Awda"/>
+               </bibl>
+            </listBibl>
          </div>
-        <div type="edition">
-           <div type="textpart" xml:id="tables">
+         <div type="edition">
+            <div type="textpart" xml:id="tables">
                <label>Sixteen circular tables, each subdivided into sixteen parts with questions on various topics</label>
-           </div>
+            </div>
             <div type="textpart" xml:id="seas">
                <label>Sixteen "seas" (<foreign xml:lang="gez">ባሕር፡</foreign>), each containing 16 sentences as a response to the previous questions</label>
+               <note>The following incipit and explicit are taken from <ref type="mss" corresp="BAVet119"/>.</note>
```

Comment:
Is it based on the manuscript or on Grébaut-Tisserant's catalogue description? If the latter, I think it's better to say this explicitly